polyfill/lib/ecmascript.mjs

```diff
@@ -5177,7 +5164,7 @@ export function RoundTime(
     case 'nanosecond':
       quantity = quantity.multiply(1000).plus(nanosecond);
   }
-  const nsPerUnit = unit === 'day' ? dayLengthNs : nsPerTimeUnit[unit];
+  const nsPerUnit = unit === 'day' ? 86400e9 : nsPerTimeUnit[unit];
```

Comment:
Could probably just fold this into a new entry in `nsPerTimeUnit` if it isn't variable now.